### PR TITLE
Cleaning up codes, TODOs, and language.

### PR DESCRIPTION
**Pull request by kisaacs**
- Remove finished TODO markers per discussion
- Fix formatting issues in MM-2
- Change "People" to "Participants" in codes
- Reword some codes into (hopefully shorter) sentences.

### CODE REVIEW

**Comment by kisaacs** [comment]
I wasn't sure what to do with the aggregative marks codes. I'm not sure I understand that what we want to say there in sentence form.

File: codes.md [hunk 1]
```diff
@@ -311,7 +304,7 @@ These codes describe observations about mental models and their formation.
     - [Session 022](https://osf.io/psv9g/) - basket drawn for carrying
 
 
-### MM6. Physical objects can also be used to represent data
+### MM6. Physical objects can be used to represent data.
   - This code cross-references mental models, depictions, AND what-is-data,
```
Comment (kawilliams) [inline codes.md:308]: Maybe "this code cross-references the themes mental models, depictions, AND what-is-data..."

File: codes.md [hunk 2]
```diff
@@ -454,12 +442,11 @@ These codes suggest how ordering arises in the mental model
       - items drawn separately
 
 
-
 ## Purpose-Seeking
 
 In these codes, participants infer a bigger picture or task.
 
-### PS1.Participants often want to add additional information, data that isn’t there
+### PS1. Participants often want to add additional data or information.
```
Comment (kawilliams) [inline codes.md]: Tense: "wanted"

File: codes.md [hunk 3]
```diff
@@ -550,7 +535,7 @@ further embellishment.
     - [Session 009](https://osf.io/es7w6/) - originally text files labeled
       with "txt" but participant says they stop labeling because of laziness
 
-### D2. People run into constraints with their existing sketch.
+### D2. Participants ran into constraints with their existing sketch.
```
Comment (kawilliams) [inline codes.md:538]: I may want to add Session 030 and 033 -- both had difficulty picking icons/file extensions for the code files that also "went with" the existing icons/encoding. Possibly also people in the junk drawer who drew pens and pencils and then were concerned about differentiating sharpies. 

This is a Katy TODO tomorrow (add 2-3 instances of each of those)

**Comment by kawilliams** [comment]
I added my tweaks and support to D2 (run into constraints). Merging, and leaving the note about possibly combining D4 (add details) and D6 (elide details)

**Pull request merged**